articles/containerization-and-why-to-use-it.md

```diff
@@ -0,0 +1,58 @@
+---
+authorName: Florian Besser
+authorGithubUsername: florian-besser
+issue: 76
+title: Containerization and why to use it
+---
+# {{page.title}}
+
+You are a good software engineer, but you dread releases as they are known as a time of pain. Your project matches one or more of the following:
```

Comment:
der Einstieg ist sprachlich grad etwas holprig:
- "dread" musste ich nachschlagen, "fear" hätte ich vertstanden, ist aber wohl zu stark?
- zweimal "as"
- Vorschlag: 
-> You are generally quite happy with your work as a software engineer, but these releases are just painful? Your project probably matches ...

Reply:
Using "you" is ok, IMHO. It's not the only article written in direct speech. @dola